ui/apps/platform/src/Containers/Collections/CollectionsTable.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { Button, ButtonVariant, Truncate, Bullseye, Text } from '@patternfly/react-core';
+import { SearchIcon } from '@patternfly/react-icons';
+import { TableComposable, TableVariant, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+
+import EmptyStateTemplate from 'Components/PatternFly/EmptyStateTemplate';
+import LinkShim from 'Components/PatternFly/LinkShim';
+import useTableSelection from 'hooks/useTableSelection';
+import { CollectionResponse } from 'services/CollectionsService';
+import { collectionsPath } from 'routePaths';
+
+export type CollectionsTableProps = {
+    collections: CollectionResponse[];
+    hasWriteAccess: boolean;
+};
+
+function CollectionsTable({ collections, hasWriteAccess }: CollectionsTableProps) {
+    const history = useHistory();
+    const { selected, allRowsSelected, onSelect, onSelectAll } = useTableSelection(collections);
+
+    function onEditCollection(id: string) {
+        history.push({
+            pathname: `${collectionsPath}/${id}`,
+            search: 'action=edit',
+        });
+    }
+
+    function onCloneCollection(id: string) {
+        history.push({
+            pathname: `${collectionsPath}/${id}`,
+            search: 'action=clone',
+        });
+    }
+
+    return (
+        <>
+            <TableComposable variant={TableVariant.compact}>
+                <Thead>
+                    <Tr>
+                        {hasWriteAccess && (
+                            <Th
+                                select={{
+                                    onSelect: onSelectAll,
+                                    isSelected: allRowsSelected,
+                                }}
+                            />
+                        )}
+                        <Th modifier="wrap" width={25}>
+                            Collection
+                        </Th>
+                        <Th modifier="wrap">Description</Th>
+                        <Th modifier="wrap" width={10}>
+                            In use
+                        </Th>
+                        <Th aria-label="Row actions" />
```

Comment:
Conditional rendering for **Row actions** column head like corresponding data cell?

Reply:
I tried this originally, but applied to both sides (the checkbox header cell and the row actions cell). I kept these in since the removal of the first cell caused some layout shift as data appeared/disappeared. It seemed semantically equivalent to keep these headers in since we still display the "collection/description/in use" headers despite there being no values in the table.